fluent-api/pom.xml

```diff
@@ -13,7 +13,38 @@
     <artifactId>codice-imaging-nitf-fluent-api</artifactId>
     <packaging>jar</packaging>
     <name>Codice Imaging: NITF Fluent API</name>
-    
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codice.imaging.nitf</groupId>
+            <artifactId>codice-imaging-nitf-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Right now, no.  In the PR for IMG-218, I'll be separating the fluent-api and fluent-impl into two packages and this will change.